### PR TITLE
revert default autopilot to standard gke

### PR DESCRIPTION
**Pull request by umeshkumhar**

### CODE REVIEW

File: applications/rag/workloads.tfvars [hunk 1]
```diff
@@ -20,7 +20,7 @@ subnetwork_cidr = "10.100.0.0/16"
 create_cluster    = true # Creates a GKE cluster in the specified network.
 cluster_name      = "<cluster-name>"
 cluster_location  = "us-central1"
-autopilot_cluster = true
+autopilot_cluster = false
```
Comment (andrewsykim) [inline applications/rag/workloads.tfvars:23]: also update jupyter and ray for consistency?

**Comment by andrewsykim** [comment]
As part of this change, we should also revert the release channel to REGULAR

**Comment by andrewsykim** [comment]
> As part of this change, we should also revert the release channel to REGULAR

FYI I pushed a commit to your branch to use REGULAR

**Review by imreddy13 — APPROVED** [review]

**Review by andrewsykim — APPROVED** [review]

**Pull request merged**